protocols/jabber/jabber.c

```diff
@@ -331,8 +342,10 @@ static void jabber_logout(struct im_connection *ic)
 	md5_free(&jd->cached_id_prefix);
 
 	g_free(jd->oauth2_access_token);
+	g_free(jd->oauth2_access_token);
```

Comment:
Double free as in double freedom!

(how the hell did that happen? must be some lazy conflict resolution)